### ISSUE TITLE
Unknown layer type: Interp

### ISSUE THREAD

**Issue by deshmukhavd** [wywu/LAB#52]
Hi,
Thanks for your great work.
I have build the provided custom caffe code successful, but when i tried to evaluate the train model, its giving me error.

`I0725 19:18:11.022178 30842 layer_factory.hpp:77] Creating layer hourglass_1/upper_4/interp
F0725 19:18:11.022199 30842 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Interp (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)
`

I tried it for CPU as well as GPU, but got this error always.
Please help me with this.
Thanks.

**Comment by richipower** [comment]
you need the modified caffe of this repo, your version of caffe does not have the Interp layer...
